code/src/cljs/sixsq/nuvla/ui/plugins/full_text_search.cljs

```diff
@@ -28,19 +29,33 @@
        :fx [[:dispatch change-event]]})))
 
 (defn FullTextSearch
-  [{:keys [db-path change-event] :as opts}]
+  [{:keys [db-path change-event placeholder-suffix] :as opts}]
   (dispatch [::helpers/set db-path ::change-event change-event])
   (let [tr   @(subscribe [::i18n-subs/tr])
+        search-placeholder (tr [:search])
         text @(subscribe [::helpers/retrieve db-path ::text])]
     [ui/Input
      (-> opts
-         (dissoc :db-path :change-event)
-         (assoc :placeholder (tr [:search])
+         (dissoc :db-path :change-event :placeholder-suffix)
+         (assoc :placeholder (if placeholder-suffix
+                               (str (str/replace search-placeholder #"\.\.\." " ") (str/lower-case placeholder-suffix) "...")
+                               search-placeholder)
                 :icon "search"
                 :value (or text "")
                 :on-change (ui-callback/input-callback
-                             #(dispatch [::search db-path %]))))]))
+                            #(dispatch [::search db-path %]))))]))
 
 (s/fdef FullTextSearch
         :args (s/cat :opts (s/keys :req-un [::helpers/db-path
                                             ::helpers/change-event])))
+
+(defn FullTextSearchWithFilterIndicator
+  [{:keys [additional-filters-applied] :as opts}]
+  [:div
+   [FullTextSearch (dissoc opts :additional-filters-applied)]
+    (when additional-filters-applied [:div {:style { :padding-left "4px"
```

Comment:
additional-filter is not doing anything with that component. Perhaps it's better to construct such 
[:div ...] outside of this component FullTextSearch.